### PR TITLE
Improved Documentation and fixes in components

### PR DESCRIPTION
**Pull request by Megha-Pathak**
This PR includes: 
- Improved Alerts, Avatar, Badge, Button component 
- Divided the documentation into different files for better maintainability

### CODE REVIEW

File: Components/home.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+@media screen and (max-width: 600px) {
+
+
+  .installation {
+    max-width: 100%;
+  }
+
+  .components {
+    max-width: 100%;
+  }
```
Comment (tusharkandpal) [inline Components/home.css:65]: can apply the css to both in single line

**Review by tusharkandpal — APPROVED** [review]
changes looks fine.! 👍🏻 

suggestion: can define box-shadow colors as variables too.

**Review by dev-enforced — APPROVED** [review]
The code looks good with proper variable naming of colors(this is new format for me)!! Would love to see the live preview of Coral UI with such well written code!!
Got to see this code block repeated at few instances:
```css
display:flex;
justify-content:center;
align-items:center;
```
So my query is it fine to take it out as a utility class or better to keep it in the component stylings itself? Good work @Megha-Pathak !!

**Comment by Megha-Pathak** [comment]
> The code looks good with proper variable naming of colors(this is new format for me)!! Would love to see the live preview of Coral UI with such well written code!! Got to see this code block repeated at few instances:
> 
> ```css
> display:flex;
> justify-content:center;
> align-items:center;
> ```
> 
> So my query is it fine to take it out as a utility class or better to keep it in the component stylings itself? Good work @Megha-Pathak !!

Moving it to component styling would be a good option. Thanks for the suggestion @dev-enforced

**Pull request merged**